torch/_torch_docs.py

```diff
@@ -3836,6 +3836,39 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins, *, min=None, max=None, weight=None, density=False) -> (Tensor, Tensor)
+
+Computes the histogram of the values in a tensor.
+
+Args:
+    {input}
+    bins: int or 1D Tensor. If int, defines the number of equal-width bins. If tensor,
+          defines the sequence of bin edges including the rightmost edge.
+
+Keyword args:
+    min (scalar): Defines the lower range of the bins. If not provided, defaults to input.min().
```

Comment:
How do min and max work if bins is a 1D tensor?

Reply:
Good question, min and max are ignored if bins is a tensor. Would it make sense to warn or error if the user provides a bins tensor as well as a min or max value?

Reply:
Error sounds good to me (since the input is invalid)

Reply:
I think error does make sense for such an input.

Worth mentioning though that numpy's histogram silently ignores range (min/max) if bins is an array.